editor/code_editor.cpp

```diff
@@ -1122,8 +1124,10 @@ void CodeTextEditor::convert_indent_to_spaces() {
 		indent += " ";
 	}
 
-	int cursor_line = text_editor->get_caret_line();
-	int cursor_column = text_editor->get_caret_column();
+	Vector<int> cursor_columns;
+	for (int c = 0; c < text_editor->get_caret_count(); c++) {
+		cursor_columns.push_back(text_editor->get_caret_column(c));
+	}
```

Comment:
This can be optimized by resizing the vector to `text_editor->get_caret_count()` instead of appending.